horace_core/GUI/horace.m

```diff
@@ -12,7 +12,7 @@
 %      HORACE('Property','Value',...) creates a new HORACE or raises the
 %      existing singleton*.  Starting from the left, property value pairs are
 %      applied to the GUI before horace_OpeningFunction gets called.  An
-%      unrecognized property name or invalid value makes property application
+%      unrecognised property name or invalid value makes property application
```

Comment:
actually either spelling is fine :)

Reply:
live all documentation to #803, this is big enough ticket without this.